music21/metadata/__init__.py

```diff
@@ -1809,23 +1768,6 @@ def _getPluralAttribute(self, attributeName: str) -> t.Tuple[str, ...]:
                 properties.MUSIC21_ABBREVIATION_TO_NAMESPACE_NAME[attributeName]
             )
 
-        # The following are in searchAttributes, and getattr will find them because
```

Comment:
this is a comment about the above lines (from prior commits).  Couldn't this, faster and more legibly be:

```
if attributeName in self._contents:
    ...do something with self._contents[attributeName]

try:
    attributeName = ...standardize attribute name
   ... do something with self._contents[attributeName]
except (KeyError, AttributeError) as ae:
    raise AttributeError(f'invalid attributeName: {attributeName}')
```

that way we get customElement searching in here as well, and short circuit the expensive standardization process when the correctly spelled name exists and the metadata exists.

BTW -- What is the return result for a correctly spelled standard attributeName but without information/entry in md._contents?  `('',)`, `None`, or AttributeError?

Reply:
It's None for singular attributes, and an empty tuple for plural ones.

It's interesting to expose custom names as attributes, although it would only work for those custom names that are valid Python attribute names.  For example, custom names might be things like 'myNamespace:blah', and that would never end up in an attribute name.

Also, I think the side-effect of adding (some) custom element searching is not something we want just yet (from a different discussion).

Reply:
AHH!!!! This is for attributes not for getItem -- never mind.  You're right, customElements shouldn't get a `.customName` inference.